net/dcwifi/libdcwproto/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2019 EWSI
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libdcwproto
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ewsi/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_HASH:=5bda395c648aa12eb90515c29024029738fde1a8f73a2cbc553be1c6962c2629
+PKG_MAINTAINER:=Carey Sonsino <careys@edgewaterwireless.com>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libdcwproto
+  SECTION:=libs
+  CATEGORY:=Libraries
+  SUBMENU:=Networking
+  TITLE:=Dual-Channel WiFi messaging library
+  URL:=https://www.edgewaterwireless.com
+  DEPENDS:= +kmod-macremapper
+endef
+
+define Package/libdcwproto/description
+Platform-independent C library for marshaling and serializing DCW messages
+endef
+
+export CROSS_COMPILE=$(TOOLCHAIN_DIR)/bin/$(TARGET_CROSS)
+export SUBDIRS=$(PKG_BUILD_DIR)
+
+define Build/Compile
+	$(MAKE) -C $(PKG_BUILD_DIR) \
+		ARCH="$(LINUX_KARCH)"
+endef
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(STAGING_DIR)/usr/include
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/include/*.h $(STAGING_DIR)/usr/include/
+	$(INSTALL_DIR) $(STAGING_DIR)/usr/lib
+	$(CP) $(PKG_BUILD_DIR)/src/.libs/$(PKG_NAME)*.so* $(STAGING_DIR)/usr/lib/
+endef
+
+define Package/libdcwproto/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	# Note: $(INSTALL_BIN) does not keep symlinks, so use $(CP)
+	$(CP) $(PKG_BUILD_DIR)/src/.libs/$(PKG_NAME)*.so* $(1)/usr/lib/
```

Comment:
Can this be replaced with PKG_INSTALL? I don't like seeing files being copied from hidden directories.